code/datums/gamemode/factions/vox_shoal.dm

```diff
@@ -106,14 +106,15 @@ var/list/potential_bonus_items = list(
 
 	for (var/i = 1 to 4)
 		var/chosen_one = pick(potential_bonus_items_temp)
-		potential_bonus_items_temp =- chosen_one
+		potential_bonus_items_temp.Remove(chosen_one)
```

Comment:
Not that it actually matters, but generally `-=` is preferred to `Remove()`

Reply:
It actually caused a bug. Using `-=` left, in my tests, a `null` entry in the `potential_bonus_items_temp` list, which could be picked up and resulted in empty strings in the final objective.
`.Remove()` did this properly.